packages/stacked_core/lib/src/code_generation/bottomsheets/stacked_bottomsheet.dart

```diff
@@ -0,0 +1,13 @@
+/// Described the Bottomsheet functionality to generate in the app
+class StackedBottomsheet {
```

Comment:
Can you please make it StackedBottomsheet<T> so that devs tell it what they expect to return i have added that on dialog to know what it expects to return on type safe

Reply:
What do I need to change exactly?

Reply:
@Eimen2018 type safety can be added in your PR. This scope has been covered here.